content/docs/command-reference/exp/pull.md

```diff
@@ -62,9 +63,9 @@ used.
 - `-r <name>`, `--remote <name>` - name of the `dvc remote` to pull cached files
   from.
 
-- `--run-cache` - downloads all available history of [stage runs] from the
-  `dvc remote` (to the cache only, like `dvc fetch --run-cache`). Note that
-  `dvc exp run <stage_name>` is necessary to checkout these files.
+- `--run-cache`, `--no-run-cache` - downloads all available history of [stage
```

Comment:
so, `--run-cache` is the default now? can we clarify that (that's it's pretty much no op and deprecated). If I understand the whole change correctly.

Reply:
I worried it's confusing since earlier versions will have the opposite behavior of the documented default, but I just made a note of when the default changed. PTAL.

Reply:
I think I am slightly confused between --run-cache and --no-run-cache - can I maybe get a better understanding between what they do, and how the time of pulling/pushing will be affected if we use or not use the cache?

Reply:
Using `--run-cache` will push and pull not only the data but also the [run-cache](https://dvc.org/doc/user-guide/pipelines/run-cache), which keeps records of each pipeline stage that has been run and the hashes associated with it, so that it can be recovered rather than running a stage with the same dependencies twice. This is usually fairly lightweight and should have pretty minimal impact on push/pull times, but because of some suboptimal performance in how it is transferred, some users found it was causing push/pull to hang. Therefore, the default was changed back to `--no-run-cache` in https://github.com/iterative/dvc/pull/10494, although we missed fixing the docs for this.